### PR TITLE
order fields of contentType form

### PR DESCRIPTION
**Pull request by djamnazi**
Order the fields of the contentType Form

### CODE REVIEW

File: Backoffice/Form/Type/FieldTypeType.php [hunk 1]
```diff
@@ -74,21 +89,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
-            ->add('type', 'choice', array(
-                'choices' => $this->getChoicesType(),
-                'label' => 'open_orchestra_backoffice.form.field_type.type',
-                'attr' => array(
-                    'class' => 'content_type_change_type'
-                )
-            ))
-            ->add('listable', 'checkbox', array(
-                'label' => 'open_orchestra_backoffice.form.field_type.listable',
+            ->add('additionalParameters', "text", array(
```
Comment (itkg-nanne) [inline Backoffice/Form/Type/FieldTypeType.php]: Not the right way to do this. You should use twig to add <hr />. This formType is named oo_field_type. So you can add a twig with :
{% block oo_field_type %}
    {% spaceless %}
    	{{ form_start() }}
    	{{ form_row(form.fieldId) }}
    	{{ form_row(form.labels) }}
    	...
        <hr />
        {{ form_rest() }}
        {{ form_end() }}
    {% endspaceless %}
{% endblock %}

not totally sure for form_rest and form_end. I let you check.

File: BackofficeBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -73,6 +73,7 @@ open_orchestra_backoffice:
             searchable: Searchable
             orderable: Orderable
             orderDirection: Order direction
+            additionalParameters: Additional parameters
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/translations/messages.en.yml]: I would better speak about "specific parameters".

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 3]
```diff
@@ -284,6 +284,24 @@
...
+    {% spaceless %}
+        {{ form_row(form.fieldId) }}
+        {{ form_row(form.labels) }}
+        {{ form_row(form.type) }}
+        {{ form_row(form.listable) }}
+        {{ form_row(form.translatable) }}
+        {{ form_row(form.searchable) }}
+        {{ form_row(form.orderable) }}
+        {{ form_row(form.orderDirection) }}
+        <div class="form-group">
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: As you want to group fields, you should maybe use:
        <fieldset>
        	<legend>{{ 'open_orchestra_backoffice.form.field_type.additionalParameters'|trans }}</legend>
        	{{ form_rest(form) }}
        </fieldset>

**Comment by djamnazi** [comment]
PR updated @itkg-nanne

**Comment by alavieille** [comment]
:+1:  whit comment of @itkg-nanne

**Comment by djamnazi** [comment]
for @itkg-nanne 's comment, Raphael says he is favorable to keep : additional parameters.

**Comment by itkg-ngilain** [comment]
@djamnazi : You forgot the rebase !!!!

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 4]
```diff
@@ -284,6 +284,23 @@
...
+        {{ form_row(form.fieldId) }}
+        {{ form_row(form.labels) }}
+        {{ form_row(form.type) }}
+        {{ form_row(form.listable) }}
+        {{ form_row(form.translatable) }}
+        {{ form_row(form.searchable) }}
+        {{ form_row(form.orderable) }}
+        {{ form_row(form.orderDirection) }}
+        <fieldset>
+            <legend>{{ 'open_orchestra_backoffice.form.field_type.additionalParameters'|trans }}</legend>
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: You should add a padding-left here

**Comment by itkg-ngilain** [comment]
@djamnazi : have you check the field type modification ? It doesn't work anymore :s

**Pull request merged**